reactioncommerce/reaction-admin
# Refactor email job table to use GraphQL

Resolves #325
Impact: **major**
Type: **performance|refactor|chore**

## Issue
The current email job table pulls data from the `Emails` Meteor publication, which is outdated and performs poorly at scale.

## Solution
Pull the email jobs from the newly-implemented `emailJobs` GraphQL query from [`api-plugin-email`#6](https://github.com/reactioncommerce/api-plugin-email/pull/6).

## Breaking changes
None.

## Testing
1. Open `Settings > Email`.
2. Check that the email jobs display properly on the table.
